CMakeLists.txt

```diff
@@ -1043,9 +1043,8 @@ function(TRANSFORM_MAKEFILE_INC INPUT_FILE OUTPUT_FILE)
   string(REPLACE "$(top_srcdir)"   "\${CURL_SOURCE_DIR}" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
   string(REPLACE "$(top_builddir)" "\${CURL_BINARY_DIR}" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
 
-  string(REGEX REPLACE "\\\\\n" "�!�" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
+  string(REGEX REPLACE "\\\\\n" "" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
```

Comment:
Okay, now it is getting strange. The rule does not apply for variable definitions. The Makefile

```
AVAR = 12\
34

command2:
        echo "${AVAR}"
        echo -n "${AVAR}" | hexdump -C
```

gives me the newline converted to a space

```
$ make command2
echo "12 34"
12 34
echo -n "12 34" | hexdump -C
00000000  31 32 20 33 34                                    |12 34|
00000005
```